shell/server/pre-meteor.js

```diff
@@ -210,6 +210,25 @@ function checkMagic(buf, magic) {
   return true;
 }
 
+function serveSelfTest(req, res) {
+  inMeteor(() => {
+    if (req.method === 'GET' &&
+        req.url === '/') {
+      const content = new Buffer("Self-test OK.");
+      res.writeHead(200, {
+        'Content-Type': 'text/plain',
+        'Content-Length': content.length,
+        'Access-Control-Allow-Origin': process.env.ROOT_URL,
```

Comment:
In theory allowing all origins is an information leak that this entity is running a Sandstorm at their domain. If we don't need `*` yet, then I'm happier to limit this to just what is needed. I can allow `*` if you very much prefer, though.